Link to plugin dependency tab and encourage to use plugin bom (#7261)
jenkins-infra/jenkins.io: content/doc/developer/testing/index.adoc

Alternatively, you can use the link:/doc/pipeline/steps/pipeline-utility-steps/#unzip-extract-zip-file[`unzip`] step from the plugin:pipeline-utility-steps[Pipeline Utility Steps plugin] to copy multiple files and folders into the workspace.

First, add the plugin to your POM as a test dependency — you can find the `groupId` and `artifactId` values in the link:https://plugins.jenkins.io/pipeline-utility-steps/dependencies[plugin dependency tab]:
[source,xml]
----
<dependency>
  <groupId>org.jenkins-ci.plugins</groupId>
  <artifactId>pipeline-utility-steps</artifactId>
  <scope>test</scope>
</dependency>
----
In general, we recommend to use the managed version from the plugin bom, so there is no need to specify the version.
But if you need a specific version you can also add a specific version with: `<version>2.16.2</version>`

> Afterward you can write a test which starts by extracting that zip file.

Reviewer: Optional;.  ChatGPT suggests alternate phrasing and punctuation as 
```suggestion
Afterwards, you can write a test that begins with extracting the zip file.
```
